src/scrape_up/codeforces/user.py

```diff
@@ -0,0 +1,81 @@
+import requests
+from bs4 import BeautifulSoup
+from datetime import datetime
+import json
+
+class USER:
+    """
+    Fetches user data from CodeForces Competitive Platform
+    """
+    user_url = "https://codeforces.com/api/user.info?handles={}"
```

Comment:
Hey @surajdeotiwari | Why are you using API instead of scraping the data?
As per the goal of the project, we are building an API alternative to fetch the data. Please do not use API.

Reply:
Ok will fix it.